rolling/distribution.yaml

```diff
@@ -2773,6 +2773,21 @@ repositories:
       url: https://github.com/DFKI-NI/mir_robot.git
       version: rolling
     status: developed
+  mocap_optitrack:
+    doc:
+      type: git
+      url: https://github.com/ros-drivers/mocap_optitrack.git
+      version: foxy-devel
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/ros-drivers-gbp/mocap_optitrack-release.git
```

Comment:
For rolling, this should be released into `ros2-gbp`.  There are instructions here as well as motivation for this change: https://docs.ros.org/en/rolling/How-To-Guides/Releasing/Release-Team-Repository.html

I got to this one last, but it's likely that you will want to release for `iron` and `humble` from that repo as well, to keep it all in one place.

Reply:
@tonybaltovski I can help coordinate mirroring the release repository into ros2-gbp.

Reply:
@nuclearsandwich that would be create.  Can we just ford the current release repo?

Reply:
> Can we just fork the current release repo?

True forks in the GitHub sense aren't easy to maintain with the automation that drives the ros2-gbp org. I usually run `git clone --mirror $ORIGINAL followed by `git push --mirror $ROS2_GBP` to seed the transfer. I'll provide more details in the linked PR.

Reply:
@nuclearsandwich anything I can do here to move this forward?

Reply:
The best way to help this going forward would be to open an issue or better a PR on https://github.com/ros2-gbp/ros2-gbp-github-org/pulls

Reply:
And after that's created with a blank repository you can migrate the ros-gbp content over. But I would suggest just rereleasing via bloom to the new target and not worry about it. You can set the debian increment number slightly higher to be extra safe.